getting-started/repl/snippets-and-scaffolders.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Snippets and Scaffolders
+page_title: Snippets and Scaffolders in Telerik REPL for Blazor
+description: Explore how to use the available Snippets and Scaffolders in Telerik REPL for Blazor
+slug: blazor-repl-snippets-scaffolders
+tags: telerik,blazor,repl,snippets,scaffolders
+published: True
+position: 5
+---
+
+# Telerik REPL for Blazor Snippets and Scaffolders
+
+Telerik REPL for Blazor supports the following features for increased developer productivity:
+
+* [Snippets](#snippets) 
+* [Scaffolders](#scaffolders)
+
+## Snippets
+
+Telerik REPL for Blazor provides a list of predefined code snippets for fast UI component reference and configuration.
+
+The basic snippets are listed in the `Snippets and Scaffolders` section of the REPL Sidebar. You can browse to find the desired snippet or search for it in the integrated Seacrhbox.
+
+A complete list of the snippets for a specific component will be displayed in a popup once you start typing the component name in the editor.
+
+REPL for Blazor allows you to add the snippets in several ways:
+
+### Add button
+
+Next to each snippet listed in the `Snippets and Scaffolders` menu, you will find a `+` button. Clicking it will automatically add the snippet in the REPL editor. The snippet will be inserted where the cursor is placed. You can then additionally configure the component as desired and add the code block for the `C#` part.
```

Comment:
Prefer more straight-forward word order, in which the subject is in the beginning of the sentence.

"as desired" does not bring additional value, as it is implied anyway.

```suggestion
There is a `+` button next to each snippet in the `Snippets and Scaffolders` menu. Clicking it will automatically add the snippet at the current cursor position in the REPL editor. Then you can additionally configure the component and add the code block for the `C#` part.
```